sdks/python/apache_beam/__init__.py

```diff
@@ -94,9 +86,8 @@
   pass
 else:
   raise RuntimeError(
-      'The Apache Beam SDK for Python is only supported on Python 2.7 or '
-      'Python 3. It is not supported on Python [' + str(sys.version_info) +
-      '].')
+      'The Apache Beam SDK for Python is only supported on Python 3. '
+      'It is not supported on Python [' + str(sys.version_info) + '].')
```

Comment:
Should we raise this error (or maybe just the warning above) for <3.6?

Reply:
python_requires >= 3.6 should make it uninstallable on <3.5, but for folks like us who import sources and don't `pip install` beam a warning wouldn't hurt.